azure-functions-gradle-plugin/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Change Log
 All notable changes to the "Azure Function Plugin for Gradle" will be documented in this file.
 - [Change Log](#change-log)
+  - [1.3.0](#130)
   - [1.2.0](#120)
   - [1.1.0](#110)
   - [1.0.0](#100)
 
+## 1.3.0
+- Support create Application Insights connection while creating/updating function apps
```

Comment:
- Support provision of Application Insights when creating a new function app.
- Support detach Application Insights from a existing function app